contracts/contracts.go

```diff
@@ -0,0 +1,125 @@
+/*
+Package contracts embeds compiled NeoFS contracts and provides access to them.
+*/
+package contracts
+
+import (
+	"embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/nef"
+)
+
+const (
+	neofsDir      = "neofs"      // not deployed to FS chain.
+	processingDir = "processing" // not deployed to FS chain.
+
+	nnsDir        = "nns"
+	alphabetDir   = "alphabet"
+	auditDir      = "audit"
+	balanceDir    = "balance"
+	containerDir  = "container"
+	neofsIDDir    = "neofsid"
+	netmapDir     = "netmap"
+	proxyDir      = "proxy"
+	reputationDir = "reputation"
+
+	nefName      = "contract.nef"
+	manifestName = "manifest.json"
+)
+
+// Contract groups information about Neo contract stored in the current package.
+type Contract struct {
+	NEF      nef.File
+	Manifest manifest.Manifest
+}
+
+var (
+	//go:embed */contract.nef */manifest.json
+	_fs embed.FS
+
+	errInvalidNEF      = errors.New("invalid NEF")
+	errInvalidManifest = errors.New("invalid manifest")
+
+	fsContracts = []string{
+		nnsDir,
+		alphabetDir,
+		netmapDir,
+		auditDir,
+		balanceDir,
+		containerDir,
+		neofsIDDir,
+		proxyDir,
+		reputationDir,
+	}
+	mainContracts = []string{
+		neofsDir,
+		processingDir,
+	}
+)
+
+// GetFS returns current set of NeoFS chain contracts stored in the package.
+// They're returned in the order they're supposed to be deployed starting
```

Comment:
but the order (`fsContracts`) differs from the implemented deployment procedure, is it intentional?

Reply:
Fixed.